### DISCUSSION TITLE
AutoencoderKL training data

### DISCUSSION THREAD

**Discussion by neeek2303** [huggingface/diffusers#8304] [General]
Hi, could you please tell what data AutoencoderKL was trained on?
For example the checkpoint from official page of the model: https://huggingface.co/docs/diffusers/api/models/autoencoderkl
```url = "https://huggingface.co/stabilityai/sd-vae-ft-mse-original/blob/main/vae-ft-mse-840000-ema-pruned.safetensors"```

 I can't find this information anywhere. Thanks

**Comment by tolgacangoz** [comment]
Hi @neeek2303. You can examine [its Model Card](https://huggingface.co/stabilityai/sd-vae-ft-mse-original).
